addOns/ascanrulesBeta/src/test/java/org/zaproxy/zap/extension/ascanrulesBeta/RelativePathConfusionScanRuleUnitTest.java

```diff
@@ -58,4 +78,32 @@ void shouldReturnExpectedMappings() {
                 tags.get(CommonAlertTag.OWASP_2017_A06_SEC_MISCONFIG.getTag()),
                 is(equalTo(CommonAlertTag.OWASP_2017_A06_SEC_MISCONFIG.getValue())));
     }
+
+    @Test
+    void shouldNotAlertIfNonExistingBackupFileReturnsNon404Code() throws Exception {
```

Comment:
`Non404` → `404` unless I'm not understanding the test. Would be good to have the test case where the alert is actually raised.

Reply:
added new test